# Pytis translations
# Copyright (C) 2006-2014 Brailcom, o.p.s.
# This file is distributed under the same license as the Pytis package.
# Tomas Cerha <cerha@brailcom.org>, 2006.
#
msgid ""
msgstr ""
"Project-Id-Version: pytis 0.1\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2014-11-07 11:17+0100\n"
"PO-Revision-Date: 2014-09-25 12:48+0200\n"
"Last-Translator: Hynek Hanke <hanke@brailcom.org>\n"
"Language-Team: Czech <wiking@lists.freebsoft.org>\n"
"Language: cs\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=3; plural=((n==1) ? 0 : (n>=2 && n<=4) ? 1 : 2);\n"

#: ../lib/pytis/data/access.py:434
msgid "Access denied"
msgstr "Neoprávněný přístup"

#: ../lib/pytis/data/dbapi.py:76
msgid "Waiting for server recovery"
msgstr "Čekám na zotavení serveru"

#: ../lib/pytis/data/dbapi.py:88
msgid "Can't connect to database"
msgstr "Nemohu se připojit do databáze"

#: ../lib/pytis/data/dbapi.py:176
msgid "Database interface error"
msgstr "Chyba v databázovém rozhraní"

#: ../lib/pytis/data/dbapi.py:195
msgid "Database connection timeout"
msgstr "Vypršel časový limit pro navázání databázového spojení"

#: ../lib/pytis/data/dbapi.py:197 ../lib/pytis/data/dbapi.py:307
msgid "Database connection error"
msgstr "Chyba databázového spojení"

#: ../lib/pytis/data/dbapi.py:211 ../lib/pytis/data/pgsql.py:87
#: ../lib/pytis/data/pgsql.py:94
msgid "Database operational error"
msgstr "Chyba databázové operace"

#: ../lib/pytis/data/dbapi.py:215 ../lib/pytis/data/pgsql.py:99
msgid "Database integrity violation"
msgstr "Porušení integrity dat v databázi"

#: ../lib/pytis/data/dbdata.py:299
msgid "Too many database connections"
msgstr "Příliš mnoho databázových spojení"

#: ../lib/pytis/data/dbdata.py:707
msgid "Database error"
msgstr "Databázová chyba"

#: ../lib/pytis/data/dbdata.py:775
msgid "Invalid user name or password"
msgstr "Chybné uživatelské jméno nebo heslo"

#: ../lib/pytis/data/pgsql.py:60
msgid "Unable to connect to the database"
msgstr "Nelze se připojit k databázi"

#: ../lib/pytis/data/pgsql.py:69
msgid "Query arguments unsupported by the pgsql backend"
msgstr "Argumenty dotazu nejsou podporovány při použití pgsql backendu"

#: ../lib/pytis/data/postgresql.py:275 ../lib/pytis/data/postgresql.py:364
#: ../lib/pytis/data/postgresql.py:376
msgid "Method not implemented"
msgstr "Volána neimplementovaná metoda"

#: ../lib/pytis/data/postgresql.py:511
msgid "Characters present in data not representable in database encoding"
msgstr "Data obsahují znaky, které nelze reprezentovat v kódování databáze"

#: ../lib/pytis/data/postgresql.py:3737
msgid "Invalid database counter value"
msgstr "Chybná hodnota čítače z databáze"

#. User input validation error message.
#: ../lib/pytis/data/types_.py:162
msgid "Empty value"
msgstr "Prázdná hodnota"

#. User input validation error message.
#: ../lib/pytis/data/types_.py:164
msgid "Invalid value"
msgstr "Neplatná hodnota"

#: ../lib/pytis/data/types_.py:396
#, python-format
msgid "Wildcard matching not supported for values of type '%s'."
msgstr "Hvězdičkové výrazy nejsou podporovány pro hodnoty typu '%s'."

#. User input validation error message.
#: ../lib/pytis/data/types_.py:530
#, python-format
msgid "Minimal value is %(minimum)s"
msgstr "Minimální hodnota je %(minimum)s"

#. User input validation error message.
#: ../lib/pytis/data/types_.py:533
#, python-format
msgid "Maximal value is %(maximum)s"
msgstr "Maximální hodnota je %(maximum)s"

#. User input validation error message.
#: ../lib/pytis/data/types_.py:610
#, python-format
msgid "Minimal size %(minlen)s not satisfied"
msgstr "Nedodržena minimální velikost %(minlen)s"

#. User input validation error message.
#: ../lib/pytis/data/types_.py:613
#, python-format
msgid "Maximal length %(maxlen)s characters exceeded"
msgstr "Řetězec přesahuje maximální délku %(maxlen)s znaků"

#. User input validation error message.
#: ../lib/pytis/data/types_.py:715
msgid "Not an integer"
msgstr "Není to celé číslo"

#. User input validation error message.
#: ../lib/pytis/data/types_.py:825
msgid "Invalid number"
msgstr "Není to povolené číslo"

#. User input validation error message.
#: ../lib/pytis/data/types_.py:959
#, python-format
msgid "Minimal length %(minlen)s characters not satisfied"
msgstr "Nedodržena minimální délka %(minlen)s znaků"

#. User input validation error message.
#: ../lib/pytis/data/types_.py:961 ../lib/pytis/data/types_.py:2042
#, python-format
msgid "Maximal size %(maxlen)s exceeded"
msgstr "Maximální velikost %(maxlen)s překročena"

#. User input validation error message.
#: ../lib/pytis/data/types_.py:1051
msgid "Enter the password twice to eliminate typos"
msgstr "Zadejte heslo dvakrát pro eliminování překlepů"

#. User input validation error message.
#: ../lib/pytis/data/types_.py:1054
msgid "Passwords don't match"
msgstr "Kontrolní zadání hesla neodpovídá"

#. User input validation error message.
#: ../lib/pytis/data/types_.py:1057
msgid "Invalid MD5 hash"
msgstr "Neplatná MD5 suma"

#. User input validation error message.
#: ../lib/pytis/data/types_.py:1060
msgid "Please use mix of letters and non-letters in your password"
msgstr "Použijte ve svém hesle kombinaci písmenných a nepísmenných znaků"

#. User input validation error message.
#: ../lib/pytis/data/types_.py:1122 ../lib/pytis/data/types_.py:1158
#: ../lib/pytis/data/types_.py:1194 ../lib/pytis/data/types_.py:1853
msgid "Invalid format"
msgstr "Neplatný formát"

#. User input validation error message.
#: ../lib/pytis/data/types_.py:1144
msgid "Invalid color format ('#RGB' or '#RRGGBB')"
msgstr "Formát barvy neodpovídá ('#RGB' nebo '#RRGGBB')"

#. User input validation error message.
#: ../lib/pytis/data/types_.py:1160
#, python-format
msgid "Invalid inet address mask: %(mask)s"
msgstr "Neplatná maska inet adresy: %(mask)s"

#. User input validation error message.
#: ../lib/pytis/data/types_.py:1162
#, python-format
msgid "Invalid inet address value %(addr)s"
msgstr "Neplatná hodnota inet adresy: %(addr)s"

#: ../lib/pytis/data/types_.py:1323
msgid "Invalid date or time format"
msgstr "Chybný formát data nebo času"

#: ../lib/pytis/data/types_.py:1324
msgid "Invalid date or time"
msgstr "Chybné datum nebo čas"

#: ../lib/pytis/data/types_.py:1325
msgid "Date outside the allowed range"
msgstr "Datum mimo povolený rozsah"

#: ../lib/pytis/data/types_.py:2007
msgid "Invalid boolean type input value"
msgstr "Neplatná vstupní hodnota typu boolean"

#: ../lib/pytis/data/types_.py:2088
#, python-format
msgid "Not a buffer object: %r"
msgstr "Není to objekt typu buffer: %r"

#: ../lib/pytis/data/types_.py:2224
#, python-format
msgid "Maximal pixel size %(maxsize)s exceeded"
msgstr "Maximální rozměr %(maxsize)s pixelů překročen"

#: ../lib/pytis/data/types_.py:2226
#, python-format
msgid "Minimal pixel size %(minsize)s exceeded"
msgstr "Minimální rozměr %(minsize)s pixelů nedodržen"

#: ../lib/pytis/data/types_.py:2228
#, python-format
msgid "Unsupported format %(format)s; valid formats: %(formats)s"
msgstr "Nepodporovaný formát %(format)s; platné formáty: %(formats)s"

#: ../lib/pytis/data/types_.py:2244
msgid "Invalid graphics format"
msgstr "Neplatný grafický formát"

#: ../lib/pytis/data/types_.py:2349
msgid "Invalid hierarchical value format"
msgstr "Chybný formát hierarchické hodnoty"

#: ../lib/pytis/data/types_.py:2351
msgid "One of hierarchical value items is too long"
msgstr "Jedna z položek hierarchické hodnoty je příliš dlouhá"

#: ../lib/pytis/data/types_.py:2353
msgid "One of hierarchical value items contains invalid characters"
msgstr "Jedna z položek hierarchické hodnoty obsahuje neplatné znaky"

#~ msgid "String exceeds max length %(maxlen)s characters"
#~ msgstr "Řetězec přesahuje maximální délku %(maxlen)s znaků"

#~ msgid "Minimal length %(minlen)s not satisfied"
#~ msgstr "Nedodržena minimální délka %(minlen)s"
